docs/user/adaptive-learning/adaptive-learning-admin.rst

```diff
@@ -0,0 +1,60 @@
+Manage Standardized Competencies (Administrator)
+==================================================
+
+.. contents:: Content of this document
+    :local:
+    :depth: 2
+
+.. _standardized_competency_catalog:
+
+Standardized Competencies
+-------------------------
+
+| The **Standardized Competency Catalog** is a catalog of competencies that is accessible to all instructors in an Artemis instance.
+  Instructors may view this catalog and import competencies from it into their course.
+| The competencies consist of a title, description and taxonomy. Instead of being part of courses, they are sorted under knowledge areas.
```

Comment:
```suggestion
| The competencies consist of a title, description and taxonomy. Instead of being tied to specific courses, competencies are organized under knowledge areas.
```